modules/nf-core/msisensorpro/msitumoronly/main.nf

```diff
@@ -0,0 +1,61 @@
+process MSISENSORPRO_MSITUMORONLY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/msisensor-pro:1.2.0--hfc31af2_0' :
+        'biocontainers/msisensor-pro:1.2.0--hfc31af2_0' }"
+
+    input:
+    tuple val(meta) , path(tumor), path(tumor_index), path(intervals)
+    tuple val(meta2), path (fasta)
+    tuple val(meta3), path (msisensor_baseline)
+
+    output:
+    tuple val(meta), path("${prefix}")         , emit: output_report
+    tuple val(meta), path("${prefix}_dis")     , emit: output_dis
+    tuple val(meta), path("${prefix}_all")     , emit: output_all
+    tuple val(meta), path("${prefix}_unstable"), emit: output_unstable
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args   ?: ''
+    def prefix   = task.ext.prefix ?: "${meta.id}"
```

Comment:
```suggestion
    prefix   = task.ext.prefix ?: "${meta.id}"
```

Reply:
Just because I am curious: I always see it the other way around (so that you need two times def and otherwise you run into errors). Has this changed? Is there any part of the docs you can point me to?